scripts/vip.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2015, Christopher Jerdonek
+# All rights reserved.
```

Comment:
This should also be updated to contain a link to the appropriate top-level OSI-approved license once we've checked it in.